game/graphics/renderer.h

```diff
@@ -89,6 +90,8 @@ class Renderer final {
     Tempest::ZBuffer          zbuffer, shadowMap[Resources::ShadowLayers];
     Tempest::ZBuffer          zbufferUi;
 
+    Tempest::Attachment       sceneTonemapped;
```

Comment:
can be moved into `fxaa` block, since at this point only FXAA uses this attachment